README.md

```diff
@@ -1 +1,17 @@
-# esp-rom-elfs
+# ESP32 series ROM ELF files
+
+The [Releases](https://github.com/espressif/esp-rom-elfs/releases) page of this repository is used to publish the ELF files of ROMs used by ESP32 series of chips.
+
+The releases will be updated whenever a new Espressif chip or a chip revision is released.
+
+## Copyrights and License
+
+The ROMs of ESP32 series of chips are Copyright (c) 2015-2022 Espressif Systems (Shanghai) Co. Ltd.
+
+The ROMs also include various third-party libraries. The list along with the respective licenses is available [here](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/COPYRIGHT.html#rom-source-code-copyrights).
+
+The ROM code of the currently available ESP32 series of chips is not open-source.
+
+You can use the ELF files in this repository under the terms of Apache License 2.0, provided in the file [LICENSE](LICENSE).
+
+Despite the fact that Apache license is usually applied to software in the source form, the license is also applicable to the compiled files — they are referred to in the license text as _"Object" form_. We chose to release the ROM ELF files under this license because that makes it easier to use these ELF files in the context of other Apache licensed projects.
```

Comment:
```suggestion
Despite the fact that Apache license is usually applied to software in the source form, the license is also applicable to the compiled files — they are referred to the license text as _"Object" form_. We chose to release the ROM ELF files under this license because that makes it easier to use these ELF files in the context of other Apache licensed projects.
```

Reply:
I think that 'in' was intentional... another way to put this is, "In the license text, they are referred to as 'Object form'". I'll rephrase this to make it sound less odd.